docs/contributing-analyzer.md

```diff
@@ -17,28 +17,33 @@ In general, it is best to run commands from the Visual Studio Developer Command
     - ASP.NET and web development
     - .NET desktop development
     - Visual Studio extension development
-1. Ensure to install Individual components:
+1. Ensure to install *Individual components*:
     - .NET Framework 4.8 SDK
     - .NET Framework 4.8 Targeting pack
+    - .NET Framework 4.7.2 Targeting pack
+    - .NET Framework 4.6 Targeting pack
     - .NET SDK
     - .NET Compiler Platform SDK
 1. Install also:
-    - .NET Core 3.1 SDK
-    - .NET 3.5 SDK (SP1)
+    - [.NET Core 3.1 SDK](https://dotnet.microsoft.com/en-us/download/dotnet)
+    - .NET 3.5 SDK (SP1) TODO: Add link to dotNetFx35setup.exe
+    - Install Visual Studio 2019 and check these SDKs in the *individual components* tab
+        - .NET framework 4 targeting pack
+        - .NET framework 4.5 targeting pack
 1. The following environment variables must be set:
-    - **JAVA_HOME**
-    - **MSBUILD_PATH** - path to the MSBuild.exe executable (MSBuild 16)
+    - **JAVA_HOME** (e.g. `C:\Program Files\Java\jdk-11.0.2`)
+    - **MSBUILD_PATH** - path to the MSBuild.exe executable (MSBuild 16 e.g. `C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\MSBuild.exe`)
     - **NUGET_PATH** - path to the nuget.exe executable (related to the [plugin integration tests](./contributing-plugin.md#integration-tests))
-    - **ORCHESTRATOR_CONFIG_URL** - url to orchestrator.properties file (for integration tests) in uri form (i.e. file:///c:/something/orchestrator.properties)
-    - **RULE_API_PATH** - path to folder containing the rule api jar
+    - **ORCHESTRATOR_CONFIG_URL** - url to orchestrator.properties file (for integration tests) in uri form (i.e. file:///c:/something/orchestrator.properties). TODO: where to get the file, where to place it and document to put that variable in "user" instead of "system"
+    - **RULE_API_PATH** - path to folder containing the rule api jar. The rule api jar can be found at [repox.jfrog](https://repox.jfrog.io/ui/artifactSearchResults?name=rule-api&type=artifacts)
```

Comment:
maybe use the link that is mentioned on https://xtranet-sonarsource.atlassian.net/wiki/spaces/LANG/pages/665824/How+to+implement+a+new+rule ?

i.e.
https://repox.jfrog.io/repox/sonarsource-private-releases/com/sonarsource/rule-api/rule-api/

Reply:
I haven't found any public available location where a contributor can download the rule api jar. Does anything like this even exist?

Reply:
it's an internal tool (https://github.com/SonarSource/sonar-rule-api) - the artifacts for private repos are private as well

this is why we need to run the rule api ourselves for community contributions 😞

Reply:
> it's an internal tool (https://github.com/SonarSource/sonar-rule-api) - the artifacts for private repos are private as well
> 
> this is why we need to run the rule api ourselves for community contributions 😞
Time to change that?

Reply:
Would be nice. There was an internal discussion about that with no conclusion. I'll ping it.